### ISSUE TITLE
Implementation details for ImageNet-R and VisDA

### ISSUE THREAD

**Issue by yushuntang** [mr-eggplant/SAR#6]
Hi, Dr. Niu:
Thanks for sharing this great work!
I tried to reproduce the TTA results for ImageNet-R and VisDA, but they are lower than reported in your paper.
Could you please share the pre-trained model and hyperparameters for ImageNet-R and VisDA, as in Table 10-12 in your paper?
Thank you!

**Comment by mr-eggplant** [comment]
Hi Yushun,

Thanks for your interest! And apologies for the delayed response. The graduation season is quite busy.

Pre-trained models (from timm) and hyper-parameters for ImageNet-R and VisDA-2021 are the same as for ImageNet-C: reliable selection threshold=0.4 x ln 1000;  $\rho$ for sharpness-aware training is 0.05.

Note that for VisDA-2021, we exclude the ObjectNet and ImageNet-O images, since these data consist of novel classes.

I reproduced results from Tables 10-11 via timm==0.6.12, matching the paper's results. 

I also found that using the latest 'timm' version could lead to different results of VitBase-LN, due to the VitBase-LN weight updates (while the ResNet-50-GN weights are still the same).

Hope that helps and kindly let me know if you have further questions :)

Best regards,

Shuaicheng
